### PR TITLE
add jenkins topic

### PR DESCRIPTION
**Pull request by casz**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [N/A] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Jenkins is widely used for automation and is highly extensible 

cc @kohsuke @oleg-nenashev

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/jenkins/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+created_by: Kohsuke Kawaguchi
+display_name: Jenkins
+github_url: https://github.com/jenkinsci/jenkins
```
Comment (oleg-nenashev) [inline topics/jenkins/index.md]: Should it be a GitHub organization link actually?
E.g. https://github.com/git is used in https://github.com/topics/git

Comment (casz) [reply topics/jenkins/index.md]: others linked directly to main repo, but it was a tossup. :)

Perhaps better to link to https://github.com/jenkinsci due to pinned repos.

Comment (casz) [reply topics/jenkins/index.md]:
```suggestion
github_url: https://github.com/jenkinsci
```

File: topics/jenkins/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+created_by: Kohsuke Kawaguchi
+display_name: Jenkins
+github_url: https://github.com/jenkinsci
```
Comment (damaneice) [inline topics/jenkins/index.md]: @casz Can you change the github_url to `https://github.com/jenkinsci/jenkins`?

Comment (casz) [reply topics/jenkins/index.md]: Did you see the conversation in the thread?

Comment (damaneice) [reply topics/jenkins/index.md]: O sorry I missed it. Thanks! I will merge once the build goes green.

**Pull request merged**